docs/source/workshops/bioc-2015.rst

```diff
@@ -1,18 +1,13 @@
 BioC 2015: Where Software and Biology Connect
 =============================================
 
-Give an overview here ... *in this workshop you will* ...
+Google has some pretty amazing big data computational "hammers" that they have been applying to search and video data for a long time.  In this workshop we take those same hammers and apply them to whole genome sequences.
```

Comment:
Unless we can attribute this language to someone not at Google, we might want to remove some of the hyperbole (whether valid or not).